lib/inspec/runner.rb

```diff
@@ -134,6 +134,8 @@ def render_output(run_data)
       return if @conf["reporter"].nil?
 
       @conf["reporter"].each do |reporter|
+        reporter[1][:include_backtrace] = @conf["reporter_backtrace_inclusion"] if @conf.key?(:reporter_backtrace_inclusion)
```

Comment:
These options are global to the run, not per-reporter. Inspec::Config (that is, `@conf`) is global, too - no need to make changes to `runner.rb` at all. You could just call it directly in `reporters/base.rb`

Reply:
I pushed another commit that leaves runner.rb unchanged